### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by dro-ex** [navidrome/navidrome#2629]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

Version 0.50.0

### Current Behavior

Version 0.50.0 will not boot with Docker on Rancher Desktop on a 2018 Mac running Sonoma

Rancher desktop mac mini 2018. Version: 0.50.0-SNAPSHOT (https://github.com/navidrome/navidrome/commit/53607fe11446a1bac581bc551ba740456dd4720d) doesn't boot either. Tag 0.50.0 doesn't work. The lastest working tag for me is "pr-2198"

### Expected Behavior

To boot and run

### Environment

```markdown
- OS: Sonoma Mac OS Rancher Desktop
- Browser: N/A
- Client: N/A
```

### How Navidrome is installed?

Docker

### Configuration

```toml
ND_SCANSCHEDULE: "24h"
ND_LOGLEVEL: "debug"
ND_SESSIONTIMEOUT: "24h"
# ND_BASEURL: ""
ND_TRANSCODINGCACHESIZE: "150MiB"
ND_IMAGECACHESIZE: "150MiB"
ND_ENABLESHARING: "true"
ND_ENABLETRANSCODINGCONFIG: "false"
ND_LASTFM_APIKEY: "xxx"
ND_LASTFM_SECRET: "xxx"
ND_IGNAOREDARTICLES: "The El La Los Las Le Les"
ND_COVERARTPRIORITY: "cover.*, embedded, front.*, folder.*"
ND_DEFAULTTHEME: "Spotify-ish"
ND_DEVSIDEBARPLAYLISTS: "true"
ND_UILOGINBACKGROUNDURL: "xxx"
```

### Relevant log output

```shell
Logs not created after displaying TOML contents
```

### Anything else?

Tried tags: 
"latest" - will not boot
"development" - will not boot
"0.50.0" - will not boot

The latest working tag for me is "pr-2198"

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by deluan** [comment]
Please see https://github.com/navidrome/navidrome/issues/2621#issuecomment-1817233434

TL;DR - Change the image to `ghrc.io/navidrome/navidrome:0.50.0` (or `latest`)

**Comment by dro-ex** [comment]
Will not boot
